docs/DeveloperGuide.md

```diff
@@ -182,11 +182,14 @@ The `AddCommand` class is responsible for adding new patient's information in th
 #### Specifications
 
 * AddCommand, as defined by the `AddCommand` class, contain parameters which consists of: 
-  *  `patientHospitalId` integer, 
-  * `name`, `preferredName` String with only alphabets character,
-  *  `foodPreference`, `familyCondition`, `hobby` and `tag` which are alphanumeric.
+  *  `patientHospitalId` non-negative numeric characters which uniquely identifies the patient, 
+  * `name`, `preferredName` String which contains alphanumeric characters,
+  *  `foodPreference`, `familyCondition`, `hobby` String and all kinds of characters,
+  *  `tag` which are alphanumeric.
 * `tag` field is optional in the AddCommand and can be added later on using the `AddTagsCommand`.
-* If any of the fields are repeated during the adding of patient or missing fields, error message will be thrown.
+* A patient can have more than 1 `f/FOOD_PREFERENCE`, `c/FAMILY_CONDITION` and `h/HOBBY`.
```

Comment:
change from `1` to `one` or ` a single` to standardise with the rest of the DG, where `1` would be used for indexing or examples, and when talking about the explanation, use `one` instead of confusion

Reply:
yes I agree!